drivers/hmis/app/graphql/types/base_audit_event.rb

```diff
@@ -180,9 +173,6 @@ def object_changes
         # Skip if changes are empty, or if the change is `nil=>99` or `99=>nil`. This is not meaningful to show in the UI.
         next if values.map { |v| v == 99 ? nil : v }.compact.empty?
 
-        # hide certain changes (SSN/DOB) if unauthorized
-        values = 'changed' if changed_record && !authorize_field(changed_record, key)
```

Comment:
I looked through environments to confirm that there are no current users who were hitting this behavior. E.g. no users who lack SSN/DOB access but have Audit access. This is safe to remove, and we are updating the description of the Audit perm accordingly. 👍